test_scripts/Defects/5_0/2401_SDL_behavior_in_case_mobile_app_sends_request_with_at_least_one_choiseSet_without_vrCommands_parameter_and_at_least_one_choiseSet_with_vrCommands_at_VR_ONLY_or_BOTH_mode.lua

```diff
@@ -0,0 +1,99 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2401
+--
+-- Description:
+-- SDL behavior in case mobile app sends request with at least one choiseSet without parameter and at least one choiseSet with at VR_ONLY or BOTH mode 
+-- Steps to reproduce:
+-- 1) Mobile app sends PerformInteraction with interactionMode:  VR_ONLY or BOTH and one or more (choiseSet_created_ without 
+--    and one or more choiseSet_created with) parameters.
+-- Expected SDL must:
+-- 1) Respond INVALID_DATA to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local paramsVrOnly = {
+    initialText = "StartPerformInteraction1",
+    interactionMode = "VR_ONLY",
+    interactionChoiceSetIDList = { 10 },
```

Comment:
@TMyza It is existing an overriding of table field `interactionChoiceSetIDList` value here: initially this field has value `{ 10 }` and it is overridden with value `{ 11 }` in next line. 
Please fix this.

The same issue appears in `paramsBoth` table also.